src/renderer/components/sceneDetail/TextGroup.tsx

```diff
@@ -0,0 +1,137 @@
+                min={0}
+                isEnabled={true}
+                onChange={this.onChangeCaptionFontSize.bind(this)}/>
+              <SimpleTextInput
+                label="Caption Font Family"
+                value={this.props.scene.captionFontFamily}
+                isEnabled={true}
+                onChange={this.onChangeCaptionFontFamily.bind(this)}/>
+              <SimpleColorPicker
+                label="Caption Color"
+                value={this.props.scene.captionColor}
+                onChange={this.onChangeCaptionColor.bind(this)}/>
+            </div>
+            <hr/>
+            <div className="ControlSubgroup">
+              <SimpleNumberInput
+                label="Big Caption Font Size (vmin)"
+                value={this.props.scene.captionBigFontSize}
+                min={0}
+                isEnabled={true}
+                onChange={this.onChangeCaptionBigFontSize.bind(this)}/>
+              <SimpleTextInput
+                label="Big Caption Font Family"
+                value={this.props.scene.captionBigFontFamily}
+                isEnabled={true}
+                onChange={this.onChangeCaptionBigFontFamily.bind(this)}/>
+              <SimpleColorPicker
+                label="Big Caption Color"
+                value={this.props.scene.captionBigColor}
+                onChange={this.onChangeCaptionBigColor.bind(this)}/>
+            </div>
+          </div>
+        )}
+      </ControlGroup>
+    );
+  }
+
+  toggleFontOptions() {
+    this.setState({showFontSettings: !this.state.showFontSettings});
+  }
+
+  update(fn: (scene: Scene | SceneSettings) => void) {
+    this.props.onUpdateScene(this.props.scene, fn);
+  }
+
+  onChangeTextKind(kind: string) { this.update((s) => { s.textKind = kind; }); }
+  onChangeTextSource(textSource: string) { this.update((s) => { s.textSource = textSource; }); }
+
+  onChangeBlinkColor(blinkColor: string) { this.update((s) => { s.blinkColor = blinkColor; }); }
+  onChangeBlinkFontSize(blinkFontSize: number) { this.update((s) => { s.blinkFontSize = blinkFontSize; }); }
+  onChangeBlinkFontFamily(blinkFontFamily: string) { this.update((s) => { s.blinkFontFamily = blinkFontFamily; }); }
+
+  onChangeCaptionColor(captionColor: string) { this.update((s) => { s.captionColor = captionColor; }); }
+  onChangeCaptionFontSize(captionFontSize: number) { this.update((s) => { s.captionFontSize = captionFontSize; }); }
+  onChangeCaptionFontFamily(captionFontFamily: string) { this.update((s) => { s.captionFontFamily = captionFontFamily; }); }
+
+  onChangeCaptionBigColor(captionBigColor: string) { this.update((s) => { s.captionBigColor = captionBigColor; }); }
+  onChangeCaptionBigFontSize(captionBigFontSize: number) { this.update((s) => { s.captionBigFontSize = captionBigFontSize; }); }
+  onChangeCaptionBigFontFamily(captionBigFontFamily: string) { this.update((s) => { s.captionBigFontFamily = captionBigFontFamily; }); }
```

Comment:
If you get tired of writing these, you could always do this:

```
changeKey(key: string, value: Any) {
  this.update((s) => s[key] = value);
}

render() {
  <div onClick={this.changeKey.bind(this, 'captionColor')}>...</div>
}

Reply:
Done